cmd/osv-scanner/main_test.go

```diff
@@ -144,438 +91,156 @@ func testCli(t *testing.T, tc cliTestCase) {
 	ec := run(tc.args, stdoutBuffer, stderrBuffer)
 	// ec := run(tc.args, os.Stdout, os.Stderr)
 
-	stdout := normalizeRootDirectory(t, normalizeFilePaths(t, stdoutBuffer.String()))
-	stderr := normalizeRootDirectory(t, normalizeFilePaths(t, stderrBuffer.String()))
+	stdout := normalizeErrors(t, normalizeTempDirectory(t, normalizeRootDirectory(t, normalizeFilePaths(t, stdoutBuffer.String()))))
```

Comment:
This is getting a bit long, can we make a "normalize" function where we compose them together.

Reply:
yeah good point I had that on my todo for refactoring but forgot about it